src/Game.java

```diff
@@ -0,0 +1,316 @@
+        }
+        askAboutLand();
+        askAboutPlant();
+        days++;
+        changeDay();
+    }
+
+    /**
+     * This method asks the player if they'd like to unlock a new piece of land.
+     * The question only pops up if they have the money to buy land,
+     * and it will stop popping up entirely when there is no more land to unlock (check lockCounter)
+     * */
+    public void askAboutLand() {
+        if (days % 10 == 0) {
+            scanner = new Scanner(System.in);
+
+            if (character.getMoney() >= 100 && lockCounter <= 6) {
+                System.out.println("Do you wanna unlock land(Yes/No)");
+                String unlockResponse = scanner.nextLine();
+                if (unlockResponse.equalsIgnoreCase("Yes")) {
+                    environment.getLands().get(lockCounter + 3).unlock();
+                    lockCounter++;
+                    character.subtractMoney(100);
+                    changeMoney();
+                } else if (unlockResponse.equalsIgnoreCase("No")) {
+                    return;
+
+                }
+            }
+        }
+    }
+
+    /**
+     * This method asks the user if they'd like to unlock new plants.
+     * The question occurs every five days.
+     */
+    public void askAboutPlant() {
+        if(!appleLock && !orangeLock && !cabbageLock){
+            return;
+        }
+        else if(days % 5 == 0) {
+            scanner = new Scanner(System.in);
+            System.out.println(
+                "Do you wanna unlock new seeds? \nApple = 50 coins (Enter A to unlock) \nOrange = 70 coins (Enter O to unlock)\nCabbage = 90 coins (Enter C to unlock)");
+            System.out.println("To skip, enter S");
+                String unlockResponse = scanner.nextLine();
+            if (unlockResponse.equalsIgnoreCase("A") && appleLock) {
+                if (character.getMoney() >= 50){
+                    appleLock = false;
+                    character.subtractMoney(50);
+                    changeMoney();
+                }
+                if (character.getMoney()<50){
+                    System.out.println("You're broke.");
+                }
+            }
+            if (unlockResponse.equalsIgnoreCase("O") && orangeLock) {
+                if (character.getMoney() >= 70){
+                    orangeLock = false;
+                    character.subtractMoney(70);
+                    changeMoney();
+                }
+                if (character.getMoney()<70){
+                    System.out.println("You're broke.");
+                }
+            }
+            if (unlockResponse.equalsIgnoreCase("C") && cabbageLock) {
+                if (character.getMoney() >= 90){
+                    cabbageLock = false;
+                    character.subtractMoney(90);
+                    changeMoney();
+                }
+                if (character.getMoney()<90){
+                    System.out.println("You're broke.");
+                }
+            }
```

Comment:
Lots of duplication here. This should be one piece of code that maps from keystroke to a `Plant` object, then asks that object for its upgrade price. Notice how many different places in your code you’re repeated the price of each plant.